Fixed remarks for Feature/hadouken-integration

phrusher opened the pull request:
I guess the branches are a bit out of sync after you squashed those commits? Let me know if there is anything I could do to ease the merge.

vktr commented:
Didn't know you had more changes, sorry :) If there aren't alot of changes then it might be easier to re-apply just those changes on top of the feature branch.

phrusher commented:
There were just the corrections based on the comments from Sonarr. I squashed the corrections into my previous commit. Not sure it was the correct way to do it? I rebased and squashed your commits just like you've done, hoping it would make it a bit easier.

phrusher commented:
Just noticed that your branch was rebased on a slightly newer commit from Sonarr so I rebased again to reduce the differences. Shouldn't be to much friction to merge now. I hope :)
